### PR TITLE
add contact- and table service

### PR DESCRIPTION
**Pull request by ChristianMoll**
data table now able to search
@plentymarkets/team-terra

### CODE REVIEW

File: src/app/services/contact.service.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{
+    constructor(private http:HttpClient)
+    {
+    }
+
+    public getContacts():Observable<TerraPagerInterface<ContactInterface>>
+    {
+        let url:string = 'http://master.login.plentymarkets.com/rest/accounts/contacts';
+
+        return this.http.get(url).pipe(map((response:TerraPagerInterface<ContactInterface>) =>
```
Comment (pweyrich) [inline src/app/services/contact.service.ts]: Wieso pipe map?

File: src/app/services/basic-table.service.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    public requestTableData():Observable<TerraPagerInterface<ContactInterface>>
+    {
+        return this.contactService.getContacts();
+    }
+
+    public dataToRowMapping(rowData:ContactInterface):TerraDataTableRowInterface<ContactInterface>
+    {
+        let cellList:Array<TerraDataTableCellInterface> = [
```
Comment (MFrank84) [inline src/app/services/basic-table.service.ts]:
```suggestion
        const cellList:Array<TerraDataTableCellInterface> = [
```

File: src/app/services/contact.service.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+@Injectable()
+export class ContactService
+{
+    constructor(private http:HttpClient)
+    {
+    }
+
+    public getContacts():Observable<TerraPagerInterface<ContactInterface>>
+    {
+        let url:string = 'http://master.login.plentymarkets.com/rest/accounts/contacts';
```
Comment (MFrank84) [inline src/app/services/contact.service.ts]:
```suggestion
        const url:string = 'http://master.login.plentymarkets.com/rest/accounts/contacts';
```

File: src/app/services/contact.service.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { ContactInterface } from '../interfaces/contact.interface';
+import { Observable } from 'rxjs/internal/Observable';
+import { TerraPagerInterface } from '@plentymarkets/terra-components';
+
+
```
Comment (MFrank84) [inline src/app/services/contact.service.ts]:
```suggestion
```

**Pull request merged**